src/main/resources/application.conf

```diff
@@ -204,6 +204,13 @@ mantis {
 
     # Extra data to add to mined blocks
     header-extra-data = "mantis"
+
+    # If true this node will mine new blocks
+    mining-enabled = false
+
+    ethash-dir = ${user.home}"/.ethash"
```

Comment:
I don't know how other clients handle it but maybe it should be a subdirectory of our `datadir` ? As a user I would probably be surprised seeing this unknown new folder in my home dir.

Reply:
It's a standard dir for DAG, these files can be shared across clients, see: https://github.com/ethereum/wiki/wiki/Ethash-DAG